.github/workflows/release.yml

```diff
@@ -175,6 +180,13 @@ jobs:
           echo "ASSET_NAME=$asset_name" >> $GITHUB_ENV
           XZ_OPT=-9 tar -C ./target/release/ -cJf $asset_name pewpew
 
+      - name: Compress for macOS
+        if: matrix.os == 'macos-latest-xlarge'
+        run: |
+          asset_name="pewpew-$VERSION-apple-darwin-arm64.tar.xz"
```

Comment:
I think we should name the binary aarch64-apple-darwin

Reply:
@summersjc Done